core/server/src/main/java/alluxio/master/file/meta/InodeTree.java

```diff
@@ -510,6 +512,12 @@ public void addInodeFromJournal(JournalEntry entry) {
 
       if (directory.getName().equals(ROOT_INODE_NAME)) {
         // This is the root inode. Clear all the state, and set the root.
+        if (SecurityUtils.isSecurityEnabled(MasterContext.getConf())
+            && mRoot != null && !mRoot.getUserName().equals(directory.getUserName())) {
+          // user is not the owner of journal root entry
+          throw new IOException(
```

Comment:
Yes, we have a `AccessControlException` for PERMISSION related exception. But here the upper methods already expect an `IOException`, and throw it if exception happens during starting a cluster. So I am not sure whether to introduce another exception in these methods call?